README.md

```diff
@@ -1,5 +1,7 @@
 #Monkey Phone
 
+[![Join the chat at https://gitter.im/codingforeveryone/MonkeyPhone](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/codingforeveryone/MonkeyPhone?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
Its a *good* idea to add the Gitter Badge. I don't know if having a specific channel/room for the MonkeyPhone project will lead to it being lost in the list (*ocean*) of Gitter channels many of us already have... by contrast, using the "*[Main](https://gitter.im/codingforeveryone?utm_source=share-link&utm_medium=link&utm_campaign=share-link)*" coding-for-everyone room is quite well frequented... 
but... maybe it makes sense to have *many* subject-specific rooms to focus the conversation...?
It can always be revised later if people decide its not being sufficiently frequented.